index.js

```diff
@@ -114,28 +135,24 @@ document.addEventListener("DOMContentLoaded", () => {
       createModel(entity, model.getAttribute("src"));
     });
   });
-      // Add click event listener
-      foundOverlay.addEventListener('click', () => {
-        // Redirect to index.html
-        window.location.href = redirectUrl;
-      });
+
+  // Add click event listener
+  foundOverlay.addEventListener("click", () => {
+    // Redirect to index.html
+    window.location.href = redirectUrl;
+  });
 
   function createModel(parent, src) {
     let newModel = document.createElement("a-gltf-model");
     newModel.setAttribute("rotation", "0 0 0");
     newModel.setAttribute("position", "0 0 0");
 
-    // Get targetIndex from parent entity
-    let targetIndex = parseInt(
-      parent.getAttribute("mindar-image-target").split(":")[1].trim(),
```

Comment:
The error was occurring here.  `mindar-image-target` was not coming in as a string on Google Chrome, this suggests the API is unreliable and perhaps best to not rely on DOM parsing in this way.